examples/credential-registry/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[package]
+name = "credential_registry"
+version = "0.1.0"
+edition = "2021"
+license = "MPL-2.0"
+authors = [ "Concordium <developers@concordium.com>" ]
+description = "An example credential registry smart contract"
+
+[features]
+default = ["std", "crypto-primitives", "wee_alloc"]
+std = ["concordium-std/std", "concordium-cis2/std"]
+wee_alloc = ["concordium-std/wee_alloc"]
+crypto-primitives = ["concordium-std/crypto-primitives"]
+
+[dependencies]
+concordium-std = {path = "../../concordium-std", version="6.2", features = ["concordium-quickcheck"], default-features = false}
+concordium-cis2 = {path = "../../concordium-cis2", version="3.0", default-features = false}
+quickcheck = {version = "1.0.3"}
```

Comment:
Is there a reason for supplying the patch version?

Reply:
No, not really. I'll fix it to use version = 1.